src/domain/service/PurchaseLottoService.js

```diff
@@ -0,0 +1,64 @@
+import CONDITION from '../../constant/Condition.js';
+import ERROR from '../../constant/Error.js';
+import LottoNumberList from '../entity/LottoNumberList.js';
+
+class PurchaseLottoService {
+  #purchaseCount;
+  #lottos;
+
+  constructor(purchaseMoneyString) {
+    this.#validate(purchaseMoneyString);
+    this.#purchaseCount = this.#calcPurchaseCount(purchaseMoneyString);
+    this.#makeLottos();
+  }
+
+  #validate(moneyString) {
+    this.#validateNotNumber(moneyString);
+    this.#validateMultiple(moneyString);
+  }
+
+  #validateNotNumber(moneyString) {
```

Comment:
왠만하면 함수명이나 변수명, 표현식에 부정표현은 쓰지 않는편이 가독성측면에서 좋아요.
부정표현을 쓰면 읽는이가 두번 생각해야하니까요? 🤔 
```suggestion
  #validateNumber(moneyString) {
```

Reply:
그렇군요 네이밍에 대한 공부를 조금 했었는데 이러한 부분에 대해서 추가로 알게되니 도움이 많이 되는 것 같습니다! 피드백 감사합니다!